Lesson4/test_fibonacci_numbers.py

```diff
@@ -0,0 +1,53 @@
+# test_fibonacci_numbers.py
+
+import pytest
+
+
+# function to be tested - from Lesson3
+def fib(n):
+    if n is None:
```

Comment:
To samo co `test_distance.py` co do flow programu - najpierw powinnaś sprawdzić warunki, a potem robić rzeczy.